sound/soc/sof/sof-audio.c

```diff
@@ -396,6 +396,9 @@ sof_unprepare_widgets_in_path(struct snd_sof_dev *sdev, struct snd_soc_dapm_widg
 	const struct sof_ipc_tplg_widget_ops *widget_ops;
 	struct snd_soc_dapm_path *p;
 
+	if (is_virtual_widget(sdev, widget, __func__))
+		return;
+
```

Comment:
is this correct?

It could very well be that a virtual widget was inserted for the only purpose of connecting two paths. In this case we do want to have the sink path handled.

I am not sure what to make of this PR. @ranj063 that's for you to review and comment.

Reply:
@plbossart I dont think we have a case where the virtual widget is added for the purpose of connecting 2 paths. I think this change is correct

Reply:
But saying "was added for some purpose" would be handwaving. What you can say instead if "virtual widgets are added for the purpose of showing connections between aggregated DAIs in SDW topologies"

Reply:
> @plbossart I dont think we have a case where the virtual widget is added for the purpose of connecting 2 paths. I think this change is correct

what about WoV, where we had a virtual widget (or route) to tie the two branches to the same PCM device?

Reply:
but even there it was purely for the purpose of making the wov pipeline show up in the list of DAPM widgets. We can still ignore all sink paths from the cirtual widget

Reply:
> But saying "was added for some purpose" would be handwaving. What you can say instead is "virtual widgets are added for the purpose of showing connections between aggregated DAIs in SDW topologies"

Commit message updated.